http/src/test/resources/application.properties

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Why is this file introduced when `http/src/main/resources/application.properties` is also used / available?